src/main/webapp/app/course/manage/detail/course-detail-line-chart.component.ts

```diff
@@ -75,20 +76,25 @@ export class CourseDetailLineChartComponent extends ActiveStudentsChart implemen
 
     constructor(private service: CourseManagementService, private translateService: TranslateService) {
         super();
+        this.translateService.onLangChange.subscribe(() => {
+            this.updateXAxisLabel();
+        });
     }
 
     ngOnChanges() {
-        this.loading = true;
         this.amountOfStudents = this.translateService.instant('artemisApp.courseStatistics.amountOfStudents');
-        // Only use the pre-loaded stats once
-        if (this.initialStatsReceived || !this.initialStats) {
-            return;
-        }
-        this.initialStatsReceived = true;
+        this.updateXAxisLabel();
         this.determineDisplayedPeriod(this.course, this.displayedNumberOfWeeks);
-        this.createLabels();
-        this.processDataAndCreateChart(this.initialStats);
-        this.data = this.dataCopy;
+        /*
+        if the course has a start date and already ended
+        (i.e. the time difference between today and the course end date is at least one week), we show the lifetime overview by default.
+         */
+        if (this.course.startDate && this.currentOffsetToEndDate !== 0) {
```

Comment:
```suggestion
        if (this.course.startDate && !!this.currentOffsetToEndDate) {
```
I guess that should also work (but its up to you :D)